### PR TITLE
feat: unCtl 1.0.0 pack

### PR DESCRIPTION
**Pull request by yurii-skrypnyk-unskript**
### unctl v1.0.0

The **unCtl** kubernetes operator to run various checks on the cluster's resources. It is based on [unCtl cli tool](https://docs.unskript.com/unctl) and provides API and web interface wrapper for you reports.

### CODE REVIEW

File: packs/unctl-1.0.0/pack.json
Comment (vishwanaths) [inline packs/unctl-1.0.0/pack.json:1]: contributor can be the name of your company

File: packs/unctl-1.0.0/values.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  # When not found, a new namespace will be created
+  namespace: "monitoring"
+  
+  content:
+    images:
+      - image: public.ecr.aws/unskript/unctl:1.0.0
+      - image: public.ecr.aws/unskript/unctl-frontend:1.0.0
+
+charts:
+  frontend:
```
Comment (vishwanaths) [inline packs/unctl-1.0.0/values.yaml:12]: frontend: and the content following that should be child of the unctl:
charts:
   unctl:
      frontend: 
and so on.
Structure to be followed is have a child for each chart under the `charts` by `name of the chart` and have the values.yaml content under it. This way, it is possible to support multiple charts in a pack

**Review by vishwanaths — CHANGES_REQUESTED** [review]

File: packs/unctl-1.0.0/values.yaml (hunk 1 above)
Comment (yurii-skrypnyk-unskript) [reply packs/unctl-1.0.0/values.yaml:12]: in the current configuration it is not really possible to split them into child charts as for eg. `frontend` is just additional container under same deployment in the chart, so all resources are pretty much dependant.

Comment (vishwanaths) [reply packs/unctl-1.0.0/values.yaml:12]: No need to split into child charts. just that move these contents under a parent chart, which in this case is unctl. This hierarchy is needed for the changes we make in packs' values.yaml to take effect.

Comment (yurii-skrypnyk-unskript) [reply packs/unctl-1.0.0/values.yaml]: done

**Review by vishwanaths — APPROVED** [review]

**Pull request merged**